account_reconcile_oca/models/account_account_reconcile.py

```diff
@@ -36,13 +36,7 @@ def _table_query(self):
     def _select(self):
         return """
             SELECT
-                CAST(
-                    (
-                        coalesce(aml.partner_id, 0) + a.id
-                    )*(
-                        COALESCE(aml.partner_id, 0)+a.id - 1
-                    )/2 + COALESCE(aml.partner_id, 0) AS INTEGER
-                ) as id,
+                ROW_NUMBER() OVER (ORDER BY 1) as id,
```

Comment:
Why don't we use min(aml.id) :thinking: Locally worked perfectly, If you have a bigger Database to test, that would be great :smile:

Reply:
It seems to work like a charm! Not sure why I did not think of it, thanks !